index.html

```diff
@@ -736,6 +745,10 @@ <h2 class="display-4 text-center" id="data_header">Loading data...</h2>
           </ul>
           <!-- Hunter Survival -->
           <ul class="tc-box" id="tc_hunter_survival" hidden>
+            <li><span class="translate_survival">Survival</span> Guides
+              <a href="https://www.icy-veins.com/wow/survival-hunter-pve-dps-guide" target="blank">Icy Veins</a>&nbsp;/&nbsp;
+              <a href="https://www.wowhead.com/survival-hunter-guide" target="blank">Wowhead</a>
+            </li>
             <li>
               <a href="https://www.icy-veins.com/wow/survival-hunter-pve-dps-guide" target="blank">Survival Guide</a>
```

Comment:
Either the previous block needs to get the mentioned update or this one. One of both need to be deleted, though.